mtp_api/apps/security/models.py

```diff
@@ -286,6 +288,23 @@ def __str__(self):
         return '{field}={value}'.format(field=self.field, value=self.value)
 
 
+class MonitoredPartialEmailAddress(TimeStampedModel):
+    keyword = models.CharField(max_length=255, unique=True, validators=[MinLengthValidator(3)])
+
+    objects = MonitoredPartialEmailAddressManager()
+
+    class Meta:
+        ordering = ('keyword',)
+        verbose_name = 'monitored partial email address'
+        verbose_name_plural = 'monitored partial email addresses'
+
+    def __str__(self):
+        return self.keyword
+
+    def matches(self, email_address: str) -> bool:
+        return self.keyword in email_address.lower()
```

Comment:
should we lowecase `self.keyword` as well? Unless you're planning to store these in lowecase?

Reply:
it happens here: https://github.com/ministryofjustice/money-to-prisoners-api/pull/769/commits/f894533b3b96f234c6c6efd87405cd52d07b61fc#diff-3cf42ec26dec1e69b6d44a59bb6095460400d603136ea39c825fd17666fa2a86R501

Reply:
admittedly, any created through django admin could be incorrect (but nobody should mess around in there anyway)